src/test/java/com/puppycrawl/tools/checkstyle/MainTest.java

```diff
@@ -629,21 +633,24 @@ public void testPrintTreeOnMoreThanOneFile() throws Exception {
             assertEquals("", systemErr.getLog());
         });
 
-        Main.main("-t", getPath("checks/metrics"));
+        Main.main("-t", getNonPackagePath(
+                "checks/metrics"));
```

Comment:
We should be able to use input directory for this as there is more than 1 file in it.